src/gui/search/searchsortmodel.h

```diff
@@ -30,10 +30,12 @@
 #define SEARCHSORTMODEL_H
 
 #include <QSortFilterProxyModel>
+#include <QStringList>
 #include "base/utils/string.h"
 
 class SearchSortModel: public QSortFilterProxyModel
 {
+    using base = QSortFilterProxyModel;
```

Comment:
Think of it as a kind of the language keyword? The sense is the following. If base class changes, then compiler warns you in the constructor body that you do not call the constructor of  the new base class. You correct one type alias and voilà, all base virtual function calls are correct. Plus you SCM does not show any text changes where there were no semantic changes (you still call base class virtual method, no matter what base class is).